### DISCUSSION TITLE
Conflicting types of MDC labels.

### DISCUSSION THREAD

**Discussion by bacek** [argoproj/argo-workflows#12078] [General]
When I switched Argo Workflows to use JSON logging to simplify attributes extraction and filtering in Kibana/OpenSearch it basically broke all the logging.

On start, `controller` will issue message "Current Worker Numbers" with attribute "workflow" equal number of workers. Which automatically mapped to "number" type in OpenSearch. Subsequent messages for actual workflows will have attribute "workflow" equals to actual workflow name. And those messages will be reject by OpenSearch with error like `failed to parse field [workflow] of type [long] in document with id 'iFjDZIsBc7pTuVck5LUF'. Preview of field's value: 'okr-workflow-template-fe21b034-99wq9'`.

I suggest to switch 2 attributes in "Current Worker Numbers" message from "workflow" and "workflowTtl" to "workflowWorkers" and "workflowTtlWorkers" to avoid conflicts.

**Comment by terrytangyuan** [comment]
Would you like to submit a PR?

**Reply by bacek** [reply]
Sure, https://github.com/argoproj/argo-workflows/pull/12083

**Reply by terrytangyuan** [reply]
Thank you!
